_api-reference/cat/cat-segment-replication.md

```diff
@@ -0,0 +1,167 @@
+---
+layout: default
+title: CAT segment replication
+parent: CAT API
+
+nav_order: 53
+has_children: false
+---
+
+# CAT segment replication
+Introduced 2.7
+{: .label .label-purple }
+
+The CAT segment replication operation returns information about active and last completed [segment replication]({{site.url}}{{site.baseurl}}/opensearch/segment-replication/index) events on each replica shard. This API returns metrics on a shard level. The metrics provide information about how far behind the replicas are lagging with respect to the primary shard. 
```

Comment:
Another small nit, I'm confused by "This API returns metrics on a shard level". 
Will the API always return metrics on a shard level on top of the information about active and completed events on shards? Can you opt out of seeing metrics in the response (based on the examples, it will return the time comparison between the primary shard, and optionally return additional metrics)?

I would consider turning the description into a compound sentence:

```suggestion
The CAT segment replication operation returns information about active and last completed [segment replication]({{site.url}}{{site.baseurl}}/opensearch/segment-replication/index) events on each replica shard and related metrics on a shard level. These metrics provide information about how far behind the replicas are lagging with respect to the primary shard. 
```